src/applications/discharge-wizard/manifest.json

```diff
@@ -1,17 +1,18 @@
 {
-  "appName": "Claims Status",
+  "appName": "Discharge Wizard",
   "entryFile": "./discharge-wizard-entry.jsx",
   "entryName": "discharge-upgrade-instructions",
   "rootUrl": "/discharge-upgrade-instructions",
   "template": {
-    "layout": "page-react-sidebar.html",
+    "layout": "page-react.html",
     "title": "How To Apply For A Discharge Upgrade",
     "heading": "How to apply for a discharge upgrade",
     "display_title": "Discharge upgrade",
     "description": "Want to apply for a military discharge upgrade or correction? Get step-by-step instructions depending on your specific situation. If your discharge is upgraded, you'll be eligible for VA benefits you earned while serving.",
     "keywords": "upgrade discharge, military discharge upgrade, discharge upgrade",
     "order": 6,
     "spoke": "Get records",
-    "collection": "records"
+    "collection": "records",
+    "includeBreadcrumbs": true
```

Comment:
Pretty sure we can remove spoke, order, and collection now, but I'm not feeling like taking risks today 🙂